### PR TITLE
Add a link to the Kubernetes code lab.

### LINKED ISSUE #1055: Add link to Kubernetes codelab
On the Kubernetes engine cloud console link, add a link/context menu item to navigate to the codelab.

### PR DESCRIPTION
**Pull request by ILMTitan**
image

Fixes #1055

### CODE REVIEW

File: GoogleCloudExtension/GoogleCloudExtension/CloudExplorerSources/CloudConsoleLinks/ConsoleLinksRoot.cs [hunk 1]
```diff
@@ -40,44 +39,33 @@ public class ConsoleLinksRoot : TreeHierarchy, ISourceRootViewModelBase
...
         private const string NetworkServiceTierPath = "net-tier";
         private const string NetworkSecurityPath = "net-security";
         private const string StackDriverMonitoringPath = "monitoring";
         private const string StackDriverDebugPath = "debug";
-        private const string StackDriverTracePath = "traces";
         private const string StackDriverLoggingPath = "logs";
         private const string StackDriverErrorReportingPath = "errors";
-        private const string StackDriverProfilerPath = "profiler";
         private const string SourceRepositoriesPath = "code";
-        private const string EndpointsPath = "endpoints";
```
Comment (simonz130) [inline GoogleCloudExtension/GoogleCloudExtension/CloudExplorerSources/CloudConsoleLinks/ConsoleLinksRoot.cs]: Please add Endpoints back.
You can use Endpoints with ASP.NET Core. I even created a community tutorial for that a few months ago :)

File: GoogleCloudExtension/GoogleCloudExtensionUnitTests/CloudExplorerSources/CloudConsoleLinks/ConsoleLinkTests.cs [hunk 2]
```diff
@@ -14,57 +14,97 @@
...
+            const string testUrl = "url://test/url";
+
+            var objectUnderTest = new ConsoleLink(
+                Mock.Of<ICloudSourceContext>(),
+                new LinkInfo(DefaultUrl, DefaultCaption),
+                new LinkInfo(testUrl, "Test Caption"));
+            objectUnderTest.NavigateInfoCommand.Execute(null);
+
+            _browserServiceMock.Verify(b => b.OpenBrowser(testUrl));
+
```
Comment (simonz130) [inline GoogleCloudExtension/GoogleCloudExtensionUnitTests/CloudExplorerSources/CloudConsoleLinks/ConsoleLinkTests.cs]: nit: remove space

File: GoogleCloudExtension/GoogleCloudExtension/CloudExplorerSources/CloudConsoleLinks/ConsoleLinksRoot.cs (hunk 1 above)
Comment (ILMTitan) [reply GoogleCloudExtension/GoogleCloudExtension/CloudExplorerSources/CloudConsoleLinks/ConsoleLinksRoot.cs]: Readded in PR #1061.

File: GoogleCloudExtension/GoogleCloudExtensionUnitTests/CloudExplorerSources/CloudConsoleLinks/ConsoleLinkTests.cs (hunk 2 above)
Comment (ILMTitan) [reply GoogleCloudExtension/GoogleCloudExtensionUnitTests/CloudExplorerSources/CloudConsoleLinks/ConsoleLinkTests.cs]: Done.

**Pull request merged**